content/contributing/membership/once-off/contents.lr

```diff
@@ -0,0 +1,16 @@
+title: Once off
+---
+body:
+
+Memberships are our preferred way to support BeeWare, as they provide income stability over time. However, we understand that this isn't always compatible with some budgets or corporate expense processes.  If you can't (or don't want to) become a member of the BeeWare project, you can make a once-off financial contribution.
+
+If the size of a once-off contributions meets or exceeds the value of a 12 month subscription, you'll be eligible to the benefits of that subscription level. 
+
+---
+display: yes
+---
+tagline: Make a once-off contribution
+---
+url: https://gum.co/YilPM
+---
+cost: 5000.1
```

Comment:
I created a once-off membership type, so that all was content and not hardcoded text, but I am sorting on cost, so that is why. I could add some ifs to the template.